core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -739,17 +739,27 @@ public void testLoadMetadataTest() throws Exception {
         LoadMetadataOptions.defaults().setCreateAncestors(true));
 
     // TODO(peis): Avoid this hack by adding an option in getFileInfo to skip loading metadata.
-    mThrown.expect(FileAlreadyExistsException.class);
-    mFileSystemMaster
-        .createDirectory(new AlluxioURI("alluxio:/a"), CreateDirectoryOptions.defaults());
+    boolean fileAlreadyExistsExceptionSeen = false;
```

Comment:
outside catch? How can I guarantee that this exception is thrown by doing Assert.fail()?